mmv1/third_party/terraform/tests/resource_vertex_ai_tensorboard_test.go

```diff
@@ -0,0 +1,152 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccVertexAITensorboard_Full(t *testing.T) {
+	t.Parallel()
+
+	random_suffix := "tf-test-" + randString(t, 10)
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckVertexAITensorboardDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccVertexAITensorboard_Full(random_suffix),
+			},
+		},
+	})
+}
+
+func testAccVertexAITensorboard_Full(random_suffix string) string {
+	return fmt.Sprintf(`
+data "google_project" "project" {
+}
+resource "google_kms_key_ring" "keyring" {
+  name     = "keyring-%s"
+  location = "us-central1"
+}
+resource "google_kms_crypto_key" "example-key" {
```

Comment:
Please switch this test to use BootstrapKMSKeyInLocation - it looks like this might even let you switch to generating the test using an example. For example: https://github.com/GoogleCloudPlatform/magic-modules/blob/2fecbdfeccd775d922cc43849d3c118d3b63c826/mmv1/products/vertexai/terraform.yaml#L35 (I forgot this was an option previously :-) )

Using the bootstrapped crypto keys lets us limit the number of crypto keys created, which is important since they can't be deleted.

Reply:
I've updated the test to use BootstrapKMSKeyInLocation.